modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/UnpublishAPIApplication.testcase

```diff
@@ -0,0 +1,150 @@
+@component-name = "portal-headless"
```

Comment:
For the name of the *.testcase file please go with the Epic name: [UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder.testcase